src/dawn/engine_dawn.cc

```diff
@@ -15,6 +15,7 @@
 #include "src/dawn/engine_dawn.h"
 
 #include "dawn/dawncpp.h"
+#include "device_metal.h"
```

Comment:
nit: dawn/device_metal.h

Reply:
It's src/dawn/device_metal.h from this directory